games-puzzle/lpairs/lpairs-1.0.4.recipe

```diff
@@ -27,14 +27,19 @@ BUILD_REQUIRES="
 	"
 BUILD_PREREQUIRES="
 	cmd:make
-	cmd:gcc
+	cmd:g++
 	cmd:ld
 	cmd:libtoolize
+	cmd:autoreconf
 	"
+if [ "$buildArchitecture"x == "x86_64"x ] ; then
+PATCHES="lpairs-1.0.4.patchset"
+fi
 
 BUILD()
 {
 	libtoolize --force --copy --install
+	autoreconf -vfi
```

Comment:
Please don't run `autoreconf` after modifying `configure` as this command will generate a brand new `configure`, overriding our changes. It's advised to modify `configure.ac` instead

Reply:
Thanks for the review! autoreconf solved missing "../depcomp". I disabled dependency tracking instead of using autoreconf to fix it. But I don't know whether it's good or not.
Program build well and run properly for me.